server/router/ride.ts

```diff
@@ -170,14 +181,50 @@ router.put('/:id', validateUser('User'), (req, res) => {
       tag: Tag.CUSTOM,
     };
   }
-  db.update(res, Ride, { id }, body, tableName);
+  db.update(res, Ride, { id }, body, tableName, (doc) => {
+    const ride = JSON.parse(JSON.stringify(doc.toJSON()));
+    const riderId = ride.rider.id;
+    const driverId = ride.driver ? ride.driver.id : null;
+    const userId = res.locals.user.id;
+    const { userType } = res.locals.user;
+
+    const info = JSON.stringify({
+      ride,
+      change: body,
+      changedBy: {
+        userType,
+        userId,
+      },
+    });
+
+    sendToUsers(info, UserType.ADMIN);
+
+    if (userType === UserType.ADMIN) {
+      ride.driver && sendToUsers(info, UserType.DRIVER, driverId);
+      sendToUsers(info, UserType.RIDER, riderId);
+    }
+    if (userType === UserType.RIDER && ride.driver) {
+      sendToUsers(info, UserType.DRIVER, driverId);
+    }
+    if (userType === UserType.DRIVER) {
+      sendToUsers(info, UserType.RIDER, riderId);
+    }
+    res.send(ride);
+  });
 });
 
 // Create edit instances and update a repeating ride's edits field
 router.put('/:id/edits', validateUser('User'), (req, res) => {
```

Comment:
are you planning on adding repeating ride edit notifications in a separate PR, or can you do it in this one?

Reply:
Yeah, I'm not entirely sure of the full functionality of the repeating rides endpoint, so in a different PR. At that point, I would probably be abstracting this to another function defined in the notifications utility file.